src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -2238,4 +2246,137 @@ namespace winrt::Microsoft::Terminal::Control::implementation
             }
         }
     }
+
+    void ControlCore::AnchorContextMenu(const til::point viewportRelativeCharacterPosition)
+    {
+        // viewportRelativeCharacterPosition is relative to the current
+        // viewport, so adjust for that:
+        _contextMenuBufferPosition = _terminal->GetViewport().Origin() + viewportRelativeCharacterPosition;
+    }
+
+    void ControlCore::ContextMenuSelectCommand()
+    {
+        const auto& marks{ _terminal->GetScrollMarks() };
+        for (auto&& m : marks)

```

Comment:
```suggestion
        for (const auto&& m : marks)
```

Reply:
One should use `const auto&` or `auto&&`, never both :)